mir-module/src/main/resources/xsl/mods2xMetaDissPlus.xsl

```diff
@@ -786,4 +789,49 @@
     </xsl:choose>
   </xsl:template>
 
+  <xsl:template name="licence">
+    <xsl:param name="derivateID" />
+    <xsl:variable name="mods" select="metadata/def.modsContainer/modsContainer/mods:mods" />
+    <xsl:choose>
+      <xsl:when test="acl:checkPermission($derivateID,'read')">
+        <ddb:licence ddb:licenceType="access">OA</ddb:licence>
+      </xsl:when>
+      <xsl:otherwise>
+        <ddb:licence ddb:licenceType="access">nOA</ddb:licence>
+      </xsl:otherwise>
+    </xsl:choose>
+    <xsl:for-each select="$mods/mods:accessCondition[@type='use and reproduction' and @xlink:href]">
+      <xsl:variable name="licenseId" select="substring-after(@xlink:href,'#')" />
+      <xsl:variable name="licence" select="document(concat('classification:metadata:0:children:mir_licenses:',$licenseId))" />
+      <xsl:variable name="licenceScheme">
+        <xsl:choose>
+          <xsl:when test="$licence//label[@xml:lang='x-dnb-type']">
+            <xsl:value-of select="$licence//label[@xml:lang='x-dnb-type']/@text" />
+          </xsl:when>
+          <xsl:otherwise>
+            <xsl:value-of select="'otherScheme'" />
+          </xsl:otherwise>
+        </xsl:choose>
+      </xsl:variable>
+      <xsl:variable name="licenceValue">
+        <xsl:choose>
+          <xsl:when test="$licence//label[@xml:lang='x-dnb']">
```

Comment:
use x-dnb-cc for the CC Lizenz
use x-dnb-oScheme for ddb:licenceType="otherScheme"
if no x-dnb* is present use  ddb:licenceType="otherScheme" and use the label of the licence insted of id